azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participantlist/ParticipantListView.kt

```diff
@@ -144,24 +142,26 @@ internal class ParticipantListView(
     }
 
     private fun generateBottomCellItems(
-        remoteParticipantCellModels: List<ParticipantListCellModel>,
-        totalActiveParticipantCount: Int,
+        participantListViewViewModel: ParticipantListViewViewModel,
     ): MutableList<BottomCellItem> {
+        val totalActiveParticipantCount = participantListViewViewModel.totalActiveParticipantCount
         val bottomCellItemsInCallParticipants = mutableListOf<BottomCellItem>()
         val bottomCellItemsInLobbyParticipants = mutableListOf<BottomCellItem>()
         // since we can not get resources from model class, we create the local participant list cell
         // with suffix in this way
-        val localParticipantViewModel = viewModel.createLocalParticipantListCellViewModel(
-            resources.getString(R.string.azure_communication_ui_calling_view_participant_drawer_local_participant)
-        )
+        val localParticipantViewModel = participantListViewViewModel.localParticipantListCell
+        val localParticipantDisplayName = (
+            localParticipantViewModel.displayName + " " +
```

Comment:
var1 + " " + var2 should be a localizable string format itself
otherwise any RTL languages is going to display these text in the wrong order

Reply:
Can maybe be done with a horizontal linear layout with middle padding, to avoid having to send out strings for localization.